### PR TITLE
Fix support for filterOptions in filters with include block

### PR DESCRIPTION
**Pull request by ivan-shaban**
Fix for issue #23

### CODE REVIEW

File: lib/handle-filters.js [hunk 1]
```diff
@@ -62,6 +56,38 @@ function handleFilters(ast, filters, options) {
...
     }
+
+    /**
+     * Get filter's options.
+     *
+     * @param node Filter node
+     * @param mergeGlobalOptions Merge local options with global, true by default
+     * @return {{}}
+     */
+    function getAttributes(node, mergeGlobalOptions) {
```
Comment (ForbesLindesay) [inline lib/handle-filters.js:67]: Why are we adding an option for `mergeGlobalOptions`.  I don't think there's ever a time when we don't merge the global options.

**Comment by ForbesLindesay** [comment]
Does this also fix #21

**Comment by ivan-shaban** [comment]
As i see now, PL #21 solve 3 problems:
1. As i describe in my issue, that filters with include block doesn't receive global options
2. Nested filters doesn't receive local options
3. Nested filters doesn't receive global options

where my PL solve only first issue, so from technical point this PL duplicates #21 

Or, if you want i can do one more commit and add the same functionality in this PL, otherwise, i think, PL may be closed.

**Comment by ForbesLindesay** [comment]
Migrated to pugjs/pug#2679

**Pull request closed without merging**